walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -24,11 +24,26 @@
 public final class FindMeetingQuery {
   /**Return collection of timeRanges that work given the events all invited attendees are going to.*/
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+    Collection<TimeRange> withOptionalAttendees = findAvailableTimes(events, request, true);
```

Comment:
Stretch: you can also create a new class with two collections, one withOpt and one without, and populate both in one visit. So you don't need to call findAvailableTimes twice (iterate the array twice) in some cases.